.github/workflows/build_release_candidate.yml

```diff
@@ -147,7 +147,72 @@ jobs:
             svn status
             svn commit -m "Staging Java artifacts for Apache Beam ${{ github.event.inputs.RELEASE }} RC${{ github.event.inputs.RC }}" --non-interactive --username ${{ github.event.inputs.APACHE_ID }} --password ${{ github.event.inputs.APACHE_PASSWORD }} 
         
-      
+  stage_python_artifacts:
+    if: ${{github.event.inputs.STAGE_PYTHON_SOURCE == 'yes'}}
```

Comment:
We need to add `STAGE_PYTHON_SOURCE` as an input at the top of the file.

Reply:
Done (changed also to `ARTIFACTS` since it is more than source I think.